requirements.dev.txt

```diff
@@ -20,7 +20,7 @@ click==7.1.2
 colorlog==4.2.1
 cookiecutter==1.7.2
 cryptography==3.0
-dataclasses==0.7
+dataclasses==0.7;python_version<'3.7'
```

Comment:
This explains all these errors I've been getting. This is an issue for a lot of modules iirc, not just aws.

Reply:
I fixed this for many modules.
dataclasses is a backport for python3.6.
It's a standard module introduced with Python3.7